test/development/app-dir/missing-required-html-tags/index.test.ts

```diff
@@ -61,7 +63,13 @@ describe('app-dir - missing required html tags', () => {
       )
     )
 
-    await assertHasRedbox(browser)
+    await openRedbox(browser)
+    // TODO: Should show "missing tags" error
+    expect(await getRedboxDescription(browser)).toMatchInlineSnapshot(`
+      "In HTML, <p> cannot be a child of <#document>.
+      This will cause a hydration error."
+      `)
+    expect(await getRedboxTotalErrorCount(browser)).toBe(1)
```

Comment:
This doesn't have the "missing tags" warning on `canary` either. It also had a fatal error from which we didn't recover which is why the assertion original passed:

```
In HTML, <p> cannot be a child of <#document>.

HierarchyRequestError: Failed to execute 'appendChild' on 'Node': Only one element on document allowed.

NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node
```

Still an issue that we have this unhelpful error though.